tools/mason/MasonBuild.chpl

```diff
@@ -0,0 +1,174 @@
+
+use TOML;
+use Spawn;
+use FileSystem;
+use MasonUtils;
+use MasonHelp;
+
+
+
+proc masonBuild(args) {
+  var show = false;
+  var release = false;
+  var compopts: [1..0] string;
+  if args.size > 2 {
+    for arg in args[2..] {
+      if arg == '-h' || arg == '--help' {
+	masonBuildHelp();
+	exit();
+      }
+      else if arg == '--release' {
+	release = true;
```

Comment:
Will we also support `--debug`, `--test`, `--bench`, etc. ?  What is the behavior when multiple modes are thrown?